backend/src/main/kotlin/metrik/project/domain/service/githubactions/GithubActionsPipelineService.kt

```diff
@@ -126,6 +126,7 @@ class GithubActionsPipelineService(
     ): Map<String, Map<GithubActionsRun, List<Commit>>> {
         val branchCommitsMap: MutableMap<String, Map<GithubActionsRun, List<Commit>>> = mutableMapOf()
         runs.groupBy { it.branch }
+            .filter { (branch, run)  -> branch.contains("main")}
```

Comment:
Thanks for your contribution first :)

What is the purpose of this filter here? Seems this line broke the test coverage check.

Reply:
this was an accident, removed it from the commit

Reply:
uh, whats wrong with the pipeline? the push action does not find the branch. https://github.com/thoughtworks/metrik/runs/5747325125?check_suite_focus=true

Reply:
That looks a existing issue, merging it.